t/cli/plugin/test_file_logger_std_output.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+. ./t/cli/common.sh
+
+log_file=logs/stdout.log
+
+if [ -f "$log_file" ]; then
+    rm "$log_file"
```

Comment:
We can avoid using real file, like https://github.com/apache/apisix/blob/4131fbf641c02eb4209ffa9cddbe0bea247e990b/t/cli/test_validate_config.sh#L30

Reply:
Ping @e1ijah1